TramsDataApi/RequestModels/CaseActions/NTI/NoticeToImprove/CreateNoticeToImproveRequest.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using TramsDataApi.DatabaseModels;
+
+namespace TramsDataApi.RequestModels.CaseActions.NTI.NoticeToImprove
+{
+    public class CreateNoticeToImproveRequest
+    {
+		[Required]
+        public int CaseUrn { get; set; }
+        public int? StatusId { get; set; }
+        public DateTime? DateStarted { get; set; }
+        public string Notes { get; set; }
+
+        public string CreatedBy { get; set; }
+        public DateTime CreatedAt { get; set; }
+        public DateTime? UpdatedAt { get; set; }
+        public DateTime? ClosedAt { get; set; }
```

Comment:
Would you have a property for ClosedAt etc when creating a notice to improve? Is it valid to create one that is already closed?